tests/fixtures/some-includes.cpp

```diff
@@ -0,0 +1,9 @@
+#include<stdio.h>
+#	include <cstdlib>
+
+int main(int argc, char* argv[])
+{
+  #include <something>
+  return 0;
+  #include not even the right syntax, but it'll still get removed
```

Comment:
Heh. I don't think you can even have leading spaces, right? Either way, seems fine for the reducer to work on it.